arch/arm64/boot/dts/qcom/sdm625-xiaomi-daisy.dts

```diff
@@ -0,0 +1,360 @@
+/* Copyright (c) 2017,2020 The Linux Foundation. All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 and
+ * only version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/dts-v1/;
+
+#include "msm8953-pm8953.dtsi"
+#include "pmi8950.dtsi"
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Xiaomi Mi A2 Lite";
+	compatible = "qcom,msm8953-daisy", "qcom,msm8953-xiaomi", "qcom,msm8953";
+
+	qcom,msm-id = <293 0>;
+	qcom,board-id= <0x1000B 0x9>;
+
+    aliases {
+        panel = &panel;
+    };
+
+    chosen {
+        bootargs = "earlycon=ttyMSM0 console=ttyMSM0 PMOS_NO_OUTPUT_REDIRECT";
+    };
+    
+	reserved-memory {
+		/delete-node/other_ext_region@85b00000;
+
+		other_ext_region@84A00000 {
+			no-map;
+			reg = <0x0 0x84A00000 0x0 0x1900000>;
+		};
+        
+        /delete-node/ adsp_fw_region@8d600000;
+        adsp_fw_mem: adsp_fw_region@8d600000 {
+			no-map;
+			reg = <0x0 0x8d600000 0x0 0x1200000>;
+		};
+
+        
+        /delete-node/  wcnss_fw_region@8e700000;
+        wcnss_fw_mem: wcnss_fw_region@8e800000 {
+			no-map;
+			reg = <0x0 0x8e800000 0x0 0x700000>;
+		};
+
+        /*ramoops@9f00000 {
+            compatible = "ramoops";
+            reg = <0x0 0x9f00000 0x0 0x100000>;
+            record-size = <0x10000>;
+            console-size = <0x60000>;
+            ftrace-size = <0x10000>;
+            pmsg-size = <0x20000>;
+            status = "okay";
+        };*/
+	};
+};
+
+&usb3 {
+    status = "okay";
+};
+
+&uart_0 {
+    status = "okay";
+    pinctrl-names = "default";
+    pinctrl-0 = <&uart_console_active>;
+};
+
+&pronto {
+    status = "okay";
```

Comment:
This is not needed right now since its already enabled in the msm8953.dtsi file.